CMake/Dependencies/libwebsockets-CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ else()
   set(LWS_WITH_SHARED 1)
 endif()
 
+set(LWS_WITH_MBEDTLS OFF)
+if (USE_MBEDTLS)
+  set(LWS_WITH_MBEDTLS ON)
+endif()
```

Comment:
Why not if/else?

Reply:
No particular reason really. But, yea, I think if else is more readable too. I'll update it.